### ISSUE TITLE
Consider PdoSessionHandler

### ISSUE THREAD

**Issue by Guite** [zikula/core#3870]
Maybe we can utilise `Symfony\Component\HttpFoundation\Session\Storage\Handler\PdoSessionHandler` and remove several parts of our own implementation.

**Comment by craigh** [comment]
I did look into this before I implemented our own solution. The PDO handler didn't seem to work the way it needed to. Perhaps you will see it differently when you look at it.

**Comment by Guite** [comment]
Right, `PdoSessionHandler` was not appropriate for us.
But I reworked `Zikula\Bridge\HttpFoundation\DoctrineSessionHandler` in order to extend `Symfony\Component\HttpFoundation\Session\Storage\Handler\AbstractSessionHandler` in fa3b3cb2cd93f2e26bf017acc323fc3e7abd8aab. With this we also support the [SessionUpdateTimestampHandlerInterface interface](https://secure.php.net/manual/en/class.sessionupdatetimestamphandlerinterface.php).

Closing.
